src/sectionproperties/analysis/section.py

```diff
@@ -1442,38 +1442,49 @@ def assemble_torsion(
             # assemble the torsion load vector
             f_torsion[el.node_ids] += f_el
 
-            # create row index vector
-            r = np.repeat(el.node_ids, n)
-
-            # create column index vector
-            c = np.tile(el.node_ids, n)
-
-            # flatten element stiffness matrix
-            k = k_el.flatten()
-
-            # add to global arrays
-            row = np.hstack((row, r))
-            col = np.hstack((col, c))
-            data = np.hstack((data, k))
+            # # create row index vector
+            # r = np.repeat(el.node_ids, n)
+            #
+            # # create column index vector
+            # c = np.tile(el.node_ids, n)
+            #
+            # # flatten element stiffness matrix
+            # k = k_el.flatten()
+            #
+            # # add to global arrays
+            # row = np.hstack((row, r))
+            # col = np.hstack((col, c))
+            # data = np.hstack((data, k))
+
+            for node_id in el.node_ids:
+                row.extend([node_id] * n)
+            col.extend(list(el.node_ids) * n)
+            data.extend(k_el.flatten())
```

Comment:
Thanks for improving and simplifying this, happy for the old commented code to be deleted!